pyVoIP/SIP.py

```diff
@@ -341,7 +341,7 @@ def __init__(self, data: bytes):
         self.body: Dict[str, Any] = {}
         self.authentication: Dict[str, str] = {}
         self.raw = data
-        self.auth_match = re.compile(",?[a-zA-Z0-9]*=")
+        self.auth_match = re.compile('(\w+)=("\w+"|\w+)')
```

Comment:
For `("\w+"|\w+)`, unfortunately `\w` does not cover `.` or other printable characters, which makes this solution not work for @TecDroiD 

Is there a better meta character that's closer to `string.printable`? Or would we need to do something like `[\w.,:;@]+` (obviously more symbols than that would be necessary but for examples sake).